src/Entity/User.php

```diff
@@ -66,7 +73,7 @@ class User implements UserInterface
      * The username of User
      * @ORM\Column(type="string", unique=true)
      * @Serializer\Expose()
-     * @Groups({"UserDetail", "UserList", "ActivityList"})
+     * @Groups({"UserDetail", "UserList", "ActivityList", "FeedbackList"})
```

Comment:
Maybe it makes sense to have `name` and `surname` also available in these groups?